pkgs/by-name/la/lager/package.nix

```diff
@@ -0,0 +1,34 @@
+{ callPackage, stdenv, lib
+, fetchFromGitHub
+, boost, immer, zug
+, pkg-config, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lager";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "arximboldi";
+    repo = "lager";
+    rev = "v${version}";
+    hash = "sha256-KTHrVV/186l4klwlcfDwFsKVoOVqWCUPzHnIbWuatbg=";
+  };
+  buildInputs = [
+    boost
+    immer
+    zug
+  ];
+  nativeBuildInputs = [
+    cmake
+  ];
+  cmakeFlags = [
+    "-Dlager_BUILD_TESTS=OFF"
+    "-Dlager_BUILD_EXAMPLES=OFF"
+  ];
+  meta = with lib; {
+    homepage    = "https://github.com/arximboldi/lager";
+    description = "library for functional interactive c++ programs";
```

Comment:
Is this description correct? It's the same as `zug` and doesn't really match what e.g. the GitHub project description says.

(The description for `zug` also doesn't really match its GitHub project description, fwiw - both might need reviewing.)

Reply:
description has been updated.